resources/styles/book-content/questions.less

```diff
@@ -0,0 +1,13 @@
+.ost-chapter-review {
+  counter-reset: question;
```

Comment:
I think that if we start with simple numbering the teacher can tell the students: "Go to ref book, 2.2, and do exercises 1,4,5,7,22" instead of "... and do exercises 1,4 in Concept Review and 2,3 in ...."